roles/splunk_heavy_forwarder/molecule/java/molecule.yml

```diff
@@ -0,0 +1,133 @@
+---
+dependency:
+  name: galaxy
+driver:
+  name: docker
+platforms:
+  - name: splunk-hf-centos8
+    image: geerlingguy/docker-centos8-ansible:latest
+    pre_build_image: true
+    command: "/lib/systemd/systemd"
+    volumes:
+      - "/opt/splunk/var"
+  - name: splunk-hf-debian10
+    image: geerlingguy/docker-debian10-ansible:latest
+    pre_build_image: true
+    command: "/lib/systemd/systemd"
+    volumes:
+      - "/opt/splunk/var"
+  - name: splunk-hf-redhat8
+    image: geerlingguy/docker-ubi8-ansible:latest
+    pre_build_image: true
+    command: "/lib/systemd/systemd"
+    volumes:
+      - "/opt/splunk/var"
+provisioner:
+  name: ansible
+  inventory:
```

Comment:
I see the vars here again. Does it not make sense for this file and `roles/splunk_heavy_forwarder/molecule/default/molecule.yml` to share most defaults and just overwrite variables specific to the playbook, like 

`java_download_url: https://download.java.net/java/GA/jdk11/9/GPL/openjdk-11.0.2_linux-x64_bin.tar.gz`

Reply:
or is it best to over-enumerate in ansible/our open-source work?

Reply:
Good question, I didn't see an immediate solution from https://molecule.readthedocs.io/en/latest/configuration.html#id12, but I also admittedly didn't look too hard into it either. Let me experiment with a few options

Reply:
I think I have a workaround for this but I'm going to merge this in and see if I can apply that in another PR